src/Model/Contact.php

```diff
@@ -2969,7 +2969,7 @@ public static function removeFollower(array $contact)
 
 		$cdata = self::getPublicAndUserContactID($contact['id'], $contact['uid']);
 
-		DI::notification()->deleteForUserByVerb($contact['uid'], Activity::FOLLOW, ['actor-id' => $cdata['public']]);
+		DI::notification()->deleteForUserByVerb($contact['uid'], Activity::FOLLOW, ['actor-id' => $contact['public']]);
```

Comment:
`$cdata` was correct here.

Reply:
Ops, I thought this was the case here. :-/

Reply:
Okay, so how can I solve the error? `$cdata` was `bool`.

Reply:
According to the code it cannot be `false` but an empty array when no contact had been found.

Reply:
My latest PR changes it back but it won't fix the bug here.